homie-controller/src/lib.rs

```diff
@@ -5,15 +5,18 @@ use rumqttc::{
     AsyncClient, ClientError, ConnectionError, EventLoop, Incoming, MqttOptions, Publish, QoS,
 };
 use std::collections::HashMap;
+use std::num::{ParseFloatError, ParseIntError};
 use std::str;
 use std::sync::{Arc, Mutex};
+use std::time::Duration;
 use thiserror::Error;
 use tokio::task::JoinError;
 
 mod types;
-pub use types::{Datatype, Device, Node, ParseDatatypeError, ParseStateError, Property, State};
+pub use types::{Datatype, Device, Extension, Node, Property, State};
+use types::{ParseDatatypeError, ParseExtensionError, ParseStateError};
 
-const REQUESTS_CAP: usize = 10;
+const REQUESTS_CAP: usize = 100;
```

Comment:
Is this just pushing the deadlock into the middle-distance, or there something specific about the deadlock case that means that it is solved completely at 100? I guess what I mean is: could we push it into the far-distance by increasing to 1000?

Reply:
Done.